README.md

```diff
@@ -186,8 +186,6 @@ No matter if the user is a new or an existing one, Yoti will always provide her/
 
 The `profile` object provides a set of attributes corresponding to user attributes. Whether the attributes are present or not depends on the settings you have applied to your app on Yoti Dashboard.
 
-Update your Callback URL in our Yoti Dashboard and you should be good to go!
```

Comment:
Didn't think it made much sense to have this piece of text here, what do you think?